config/default.conf

```diff
@@ -0,0 +1,72 @@
+# kafka producer     
+producer.metadata.broker.list=kafka:9092
+producer.compression.codec=gzip
+producer.retry.backoff.ms:integer=200
+producer.message.send.max.retries:integer=10
+producer.socket.keepalive.enable:boolean=true
+producer.queue.buffering.max.messages:integer=100000
+producer.queue.buffering.max.ms:integer=1000
+producer.batch.num.messages:integer=1000000
+producer.dr_cb:boolean=true
+producer.enable.idempotence:boolean=false
+producer.max.in.flight.requests.per.connection:integer=1000000
+
+# kafka consumer  
+consumer.group.id=cron
+consumer.client.id=${HOSTNAME:-cron}
+consumer.metadata.broker.list=kafka:9092
+consumer.max.in.flight.requests.per.connection:integer=1000000
+consumer.socket.keepalive.enable:boolean=false
+    
+# kafka topic
+topic.acks:integer=-1
+topic.auto.offset.reset=earliest
+      
+# sdk consumer
+sdkConsumer.in.processing.max.messages:integer=1
+sdkConsumer.queued.max.messages.bytes:integer=10485760
+sdkConsumer.subscription.backoff.min.ms:integer=1000
+sdkConsumer.subscription.backoff.max.ms:integer=60000
```

Comment:
use only capital letters, config apk lib does not work in some cases as well with capital letters in this file.